typescript-test-samples/schema-and-contract-testing/tests/contract-testing.test.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import event_1_0_0 from '../events/event-1.0.0.json';
+import event_1_2_0 from '../events/event-1.2.0.json';
+import event_1_3_0 from '../events/event-1.3.0.json';
+
+// Sample business logic code - we expect to have a full address in the "address" string, e.g. "2 Park St, Sydney, NSW 2000, Australia"
```

Comment:
At first I thought this function was part of the test logic, but your comment here helped me understand that its intended to be business logic that we're using as a part of the contract test.

I'm glad to see this approach! I agree that the contract tests should use the business logic code for this type of check.

I'm wondering if there's a way to emphasize the fact that this is business/application code... maybe moving this out to its own module under a `/src` top-level directory?

Reply:
Makes sense. Updated this.